tests/sinks/property_per_sink.test

```diff
@@ -0,0 +1,27 @@
+program=lexical_scopes
+chronicles_sinks="textlines[file(lines.txt),nocolors],textblocks[file(blocks.txt),notimestamps]"
+
+[Output]
+lines.txt="""INF 2018-10-12 12:24:21+02:00 main started                               topics="main" thread=0 a=10 arg=50 b=inner-b c=10 d=some-d x=16 z=20
+INF 2018-10-12 12:24:21+02:00 exiting                                    thread=0 a=12 b=overriden-b c=100 msg="bye bye" x=16
+"""
+
+blocks.txt="""[32mINF[0m [1mmain started[0m topics="[93mmain[0m"
```

Comment:
hmm, I never considered this a bug but rather a feature :-) , but I didn't give it any thought until now either. It is indeed very handy to have the colors displayed with `cat`, `less`, ... So I would keep it.

When checking logs from an editor it will be annoying, but then again the colors option can be disabled per sink. So it will depend on the workflow of the user. Depending on how it would be used the most I guess the default setting could be changed? But then a different default for stdout vs file might also be confusing.

One feature I never tested is syslog, I don't know what happens there default but I could imagine that you do not want the ansi colors.